invenio_app_ils/ui/backoffice/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
manifest should be specific to the project, not the example react-app

Reply:
This is automatically generated by create-react-app.